zoom_rooms/indico_zoom_rooms/tasks.py

```diff
@@ -0,0 +1,130 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2024 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from pprint import pformat
+
+import requests
+from celery.schedules import crontab
+from celery.utils.log import get_task_logger
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.models.events import Event
+from indico.modules.events.sessions.models.blocks import SessionBlock
+from indico.util.string import strip_control_chars
+
+from indico_zoom_rooms.models import ZoomRoomsAction, ZoomRoomsQueueEntry
+
+
+def obj_from_locator(locator: dict) -> Contribution | SessionBlock | Event:
```

Comment:
Isn't this a simplified version of `get_object_from_args` which we already have in the core? Assuming you have full locators that contain the parent objects.

Reply:
Seems like we're not using this function anymore. I'll get rid of it.